checkstyle-sonar-plugin/pom.xml

```diff
@@ -74,6 +67,19 @@
       <artifactId>commons-lang</artifactId>
       <version>2.6</version>
     </dependency>
+    <dependency>
+      <!-- required during runtime, by sonar 5.6.4 -->
```

Comment:
please remove 5.6.4 reference, we will use 6.7 as minimal required version.
It would be better to test on latest 7.X also.